.github/workflows/boundary-service.yaml

```
@@ -0,0 +1,73 @@
+  Workflow_Trigger:
+    if: startsWith(github.ref, 'refs/heads/develop')
+    runs-on: ubuntu-latest
+    needs: docker_image-build
+    steps:
+      - name: Trigger Deployment Pipeline
+        run: |
+          curl -X POST \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: token ${{ secrets.GHUB_TOKEN }}" \
+          https://api.github.com/repos/Selco-Foundation/DIGIT-DevOps/actions/workflows/Dev.yaml/dispatches \
+          -d '{"ref":"Selco-Dev"}'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)